members/static/members/js/department_map.js

```diff
@@ -43,7 +43,8 @@ document.addEventListener("DOMContentLoaded", event => {
 function addToMap(department) {
   if (
     department.address.latitude === null ||
-    department.address.latitude === null
+    department.address.latitude === null ||
+    department.isVisible == false
```

Comment:
Til en anden gang kan du bare skrive `!department.isVisible` som læses som hvor `!` læses som not.

Reply:
Right, det ville være pænere.